### ISSUE TITLE
Graphics Glitches 

### ISSUE THREAD

**Issue by fordstone** [MultiCraft/MultiCraft-legacy#211]
Running a MacBook Pro on High Sierra with Intel HD Graphics 3000 512 MB
The game has horrible graphical glitches in the background that renders it unplayable. I am running version 2.0.0 and have tried all different graphics settings including all of the graphics drivers. 
![Screen Shot 2021-05-13 at 12 54 01 PM](https://user-images.githubusercontent.com/84148893/118165831-626d9500-b3ea-11eb-8991-96a1c2add834.png)

Any thoughts?

**Comment by MoNTE48** [comment]
The only graphics driver that works with macOS is OpenGL.
Do you see this problem with the disabled shaders?
We have tested the game on various versions of macOS and have never encountered such problems.

**Comment by fordstone** [comment]
Yes, I just tried disabling shaders and same issue. Open to any other suggestions 
![Screen Shot 2021-05-13 at 2 23 58 PM](https://user-images.githubusercontent.com/84148893/118176432-065d3d80-b3f7-11eb-96b7-ee327efbb056.png)
